BuildLibs/buildmap.py

```diff
@@ -1,153 +1,190 @@
-from struct import *
-from BuildLibs import *
-from BuildLibs import games
+import binascii
+import copy
+import random
+import struct
 from pathlib import Path
+from typing import OrderedDict, Union
+
+from BuildLibs import crc32, error, games, info, swapobjkey, trace, warning, FancyPacker
+
 
 class CStat:
+
     def __init__(self, cstat):
         self.blocking = bool(cstat & 1)
         self.facing = cstat & 0x30
         self.onesided = cstat & 0x40
         self.blockingHitscan = bool(cstat & 0x800)
         self.invisible = bool(cstat & 0x8000)
 
+
 class Sector:
-    def __init__(self, **kargs):
-        self.__dict__ = dict(**kargs)
-        self.nearbySectors:set
-        self.walls:list
-        self.shapes:list
+
+    def __init__(self, **kwargs):
+
+        self.wall: int = kwargs.get('wall', 0)
```

Comment:
I think we should rename this `wall` to `wallptr` to try to make it more obvious what it is